de.fu_berlin.inf.dpp.intellij/src/de/fu_berlin/inf/dpp/intellij/ui/menu/SarosFileShareGroup.java

```diff
@@ -68,12 +68,16 @@ public void actionPerformed(AnActionEvent e) {
             return new AnAction[0];
         }
 
+        int userCount = 0;
```

Comment:
Classic: Start counting at zero 😄

Reply:
I didn't think counting from zero is a problem here. But I can change it if you would prefer.

Reply:
Some users may be confused. Imagine you have one single contact, and this appears as your only entry: "0: bob".

This if fine for now. We can conduct user tests in the future, if we really want to know what works best.

Reply:
User count now starts at 1.